### ISSUE TITLE
Probabilités des phrases

### ISSUE THREAD

**Issue by ProgVal** [petosorus/emergencyPipou#4]
Le README dit « Créer un fichier <code>words.conf</code> des mots à utiliser, précédés de leurs probabilités (suivre l'exemple en utilisant bien des tabulations) »
Mais dans l’exemple, il n’y a pas de probabilité (et il ne me semble pas que le code le gère non plus)

**Comment by petosorus** [comment]
Oui, ça a été retiré mais j'ai oublié de changer le README. Je peux remettre une fonction qui le fait si tu veux.
